### DISCUSSION TITLE
Scroll up

### DISCUSSION THREAD

**Discussion by free3p** [idiotWu/smooth-scrollbar#534] [Q&A] [answered]
Hi. 
Can you make it so that when you refresh the page, it does not scroll up, but stays in its current place.

**Comment by sadeghbarati** [comment, marked as answer]
Hi,

Store the "offset You have scrolled" somewhere like `localStorage`

and [setPosition](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarsetposition) after page loaded

**Reply by free3p** [reply]
Thanks

**Reply by sadeghbarati** [reply]
Note that, it's better to use `debounce` for this kinda situation
